tools/Customization/DevHome.Customization/CustomControls/DevDriveOptimizerCardBody.xaml

```diff
@@ -0,0 +1,82 @@
+    </UserControl.Resources>
+
+    <Grid
+            Style="{StaticResource CardBodyContainerStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+        <!--
+                Stackpanel containing name of cache to be optimized and a description.
+            -->
+        <StackPanel
+                Grid.Column="1"
+                Grid.ColumnSpan="2"
+                Style="{StaticResource CardBodyStackPanelStyle}">
+            <StackPanel Orientation="Vertical">
+                <Grid Margin="50 10 50 10">
+                    <TextBlock Margin="5" Grid.Column="0"
+                        Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}"
+                        Text="{x:Bind CacheToBeMoved, Mode=OneWay}"
+                        IsTextSelectionEnabled="True"/>
+                    <Grid HorizontalAlignment="Right">
+                        <FontIcon HorizontalAlignment="Left" Margin="-10" Glyph="&#xE946;" FontSize="12" FontFamily="{StaticResource SymbolThemeFontFamily}"/>
+                        <TextBlock HorizontalAlignment="Left" Margin="5 5 5 5" Grid.Column="1"
+                            Foreground="PowderBlue"
+                            Text="Suggestion"
+                            IsTextSelectionEnabled="True"/>
+                    </Grid>
+                </Grid>
+                <Line Margin="-5 10 -5 0" Stroke="Black" Stretch="Fill" X1="0" Y1="0" X2="1" Y2="0" StrokeThickness="1"/>
+                <Grid Margin="50 10 10 10">
+                    <StackPanel>
+                        <TextBlock TextWrapping="Wrap" Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}">                            
+                            <Run>Global environment variable </Run>
+                            <Run Text="{Binding Path=EnvironmentVariableToBeSet, Mode=OneWay}" Foreground="PowderBlue"/>
+                            <Run> is set to </Run>
+                            <Run Text="{Binding Path=ExistingCacheLocation, Mode=OneWay}" Foreground="PowderBlue"/>
+                            <Run>, which is not located on dev drive. Move contents of this folder to a directory on dev drive such as </Run>
+                            <Run Text="D:\packages\pip\cache" Foreground="PowderBlue"/>
+                            <Run> and set </Run>
+                            <Run Text="{Binding Path=EnvironmentVariableToBeSet, Mode=OneWay}" Foreground="PowderBlue"/>
+                            <Run> to that chosen directory on dev drive</Run>
+                        </TextBlock>
+                    </StackPanel>
+                </Grid>
+                <Button Margin="50 10 10 10"
+                    Padding="25 4 25 6"
+                    MinWidth="120"
+                    Content="Make the change"

```

Comment:
`Make the change`? is this agreed upon content?

Reply:
Yes. 
[https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=2248-23987&mode=design&t=qtrgt0X2Dxli2fHU-4](https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=2248-23987&mode=design&t=qtrgt0X2Dxli2fHU-4)